modules/promotion/src/Element/CouponRedemptionForm.php

```diff
@@ -32,6 +32,9 @@ public function getInfo() {
     return [
       // The order to which the coupon will be applied to.
       '#order' => NULL,
+      '#code_label' => t('Coupon code'),
+      '#code_description' => t('Enter your coupon code here.'),
+      '#apply_label' => t('Apply coupon'),
```

Comment:
I'm not arguing against the use case at all. I just figure that if I pass a #title to the coupon redemption form element, I'd expect it to be shown above the textarea (Coupon code).